### PR TITLE
Add swagger to auth-api

### PR DESCRIPTION
**Pull request by dougwettlaufer**

### CODE REVIEW

**Comment by dougwettlaufer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- auth-api/src/main/java/io/stargate/auth/api/impl/WebImpl.java  1
- auth-api/src/main/java/io/stargate/auth/api/impl/Server.java  1
- auth-api/src/main/java/io/stargate/auth/api/AuthApiActivator.java  4
- auth-api/src/main/java/io/stargate/auth/api/resources/AuthResource.java  8
         

Clones added
============
- auth-api/src/main/java/io/stargate/auth/api/impl/Server.java  1
- restapi/src/main/java/io/stargate/web/RestApiActivator.java  1
- auth-api/src/main/java/io/stargate/auth/api/AuthApiActivator.java  1
- auth-api/src/main/java/io/stargate/auth/api/resources/AuthResource.java  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/stargate/stargate/pullRequest?prid=6316379&bid=20535507)

**Comment by dougwettlaufer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
- Added 2
           

Complexity increasing per file
==============================
- auth-api/src/main/java/io/stargate/auth/api/resources/AuthResource.java  8
- auth-api/src/main/java/io/stargate/auth/api/swagger/SwaggerUIResource.java  5
- auth-api/src/main/java/io/stargate/auth/api/impl/Server.java  1
- auth-api/src/main/java/io/stargate/auth/api/impl/WebImpl.java  1
- auth-api/src/main/java/io/stargate/auth/api/AuthApiActivator.java  4
         

Clones added
============
- restapi/src/main/java/io/stargate/web/RestApiActivator.java  1
- auth-api/src/main/java/io/stargate/auth/api/resources/AuthResource.java  7
- auth-api/src/main/java/io/stargate/auth/api/swagger/SwaggerUIResource.java  1
- auth-api/src/main/java/io/stargate/auth/api/impl/Server.java  2
- auth-api/src/main/java/io/stargate/auth/api/AuthApiActivator.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/stargate/stargate/pullRequest?prid=6316379&bid=20535507)

**Comment by dougwettlaufer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
- Added 2
           

Complexity increasing per file
==============================
- auth-api/src/main/java/io/stargate/auth/api/impl/WebImpl.java  1
- auth-api/src/main/java/io/stargate/auth/api/resources/AuthResource.java  8
- auth-api/src/main/java/io/stargate/auth/api/impl/Server.java  1
- auth-api/src/main/java/io/stargate/auth/api/swagger/SwaggerUIResource.java  5
- auth-api/src/main/java/io/stargate/auth/api/AuthApiActivator.java  4
         

Clones added
============
- auth-api/src/main/java/io/stargate/auth/api/resources/AuthResource.java  7
- auth-api/src/main/java/io/stargate/auth/api/impl/Server.java  2
- restapi/src/main/java/io/stargate/web/RestApiActivator.java  1
- auth-api/src/main/java/io/stargate/auth/api/swagger/SwaggerUIResource.java  1
- auth-api/src/main/java/io/stargate/auth/api/AuthApiActivator.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/stargate/stargate/pullRequest?prid=6316379&bid=20535507)

File: auth-api/src/main/java/io/stargate/auth/api/config/ApplicationConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright The Stargate Authors
```
Comment (EricBorczuk) [inline auth-api/src/main/java/io/stargate/auth/api/config/ApplicationConfiguration.java:2]: qq: which file(s) require this license at the top?

**Review by EricBorczuk — APPROVED** [review]

File: auth-api/src/main/java/io/stargate/auth/api/config/ApplicationConfiguration.java (hunk 1 above)
Comment (dougwettlaufer) [reply auth-api/src/main/java/io/stargate/auth/api/config/ApplicationConfiguration.java:2]: Any java file that we've created

**Pull request merged**